ngx-fudis/projects/ngx-fudis/src/lib/components/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+	selector: 'fudis-breadcrumbs',
+	templateUrl: './breadcrumbs.component.html',
+	styleUrls: ['./breadcrumbs.component.scss'],
+})
+export class BreadcrumbsComponent {
+	@Input() links: { label: string; url: string }[] = [];
```

Comment:
Separate and make new type _FudisBreadcrumb_ to `types/miscellaneous.ts` and use it here.